.github/workflows/build-lint-test.yml

```diff
@@ -11,8 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        # TODO: Test on more recent LTS versions
-        node-version: [12.x]
+        node-version: [14.x, 16.x]
```

Comment:
:+1:

Reply:
You could try adding this here, although based on other PRs in which we've tried this, it may require code changes, so if that's the case I would recommend doing this in a separate PR.